src/spark_rapids_ml/core.py

```diff
@@ -97,6 +94,12 @@
 Pred = namedtuple("Pred", ("prediction", "probability"))
 pred = Pred("prediction", "probability")
 
+# Global parameter alias used by core and subclasses.
+ParamAlias = namedtuple(
+    "ParamAlias", ("init", "handle", "num_cols", "part_sizes", "loop")
+)
+param_alias = ParamAlias("cuml_init", "handle", "n", "part_sizes", "loop")
```

Comment:
Did you intend to change "n" to "num_cols" here, since otherwise `param_alias.numl_cols` is still "n"?  Also, since we're making the change at this time, is there any reason why we shouldn't just set the values equal to the field names for now, e.g.
```
ParamAlias = namedtuple(
    "ParamAlias", ("cuml_init", "handle", "num_cols", "part_sizes", "loop")
)
param_alias = ParamAlias("cuml_init", "handle", "num_cols", "part_sizes", "loop")
```

Reply:
Yeah, "n" has been used in the argument lists of cuml multi-gpu functions. "num_cols" reads more informative but replacement may trigger errors in calling cuml multi-gpu functions. 

It seems safe to replace "init" with "cuml_init".